srv-cal-json.go

```diff
@@ -98,23 +101,28 @@ func jsonHandler(w http.ResponseWriter, r *http.Request) {
 	if err != nil {
 		log.Fatalf("Unable to retrieve next ten of the user's events: %v", err)
 	}
-    json.NewEncoder(w).Encode(events)
+	json.NewEncoder(w).Encode(events)
 }
 
 // type events map[string]string
 
 func main() {
-    http.HandleFunc("/foo", jsonHandler)
-    log.Fatal(http.ListenAndServe(":8080", nil))
-
-    // fmt.Printf("events: %s\n", events)
-    // h :=events
-    // b, err := json.Marshal(events)
-    // http.HandleFunc("/foo", b)
-    // log.Fatal(http.ListenAndServe(":8080", nil))
-    // fmt.Printf("%s\n", b)
-    // if err != nil {
-        // log.Fatalf("Eh, oops: %v", err)
-    // }
-}
```

Comment:
Lot of white space changes, but just adds the controller layer to the existing endpoints.

Reply:
Good call